lib/roku_builder/plugins/core.rb

```diff
@@ -122,12 +114,26 @@ def dounstage(options:)
       Stager.new(config: @config, options: options).unstage
     end
 
+    def self.get_help_text(parser:, options:, plugin_name:)
```

Comment:
I moved this to its own function so it can be tested --- you can't test a function easily with an "exit" call